bin/compile

```diff
@@ -42,6 +42,7 @@ mkcd()
     mkdir -p $1 && cd $1
 }
 
+echo "export PATH=${cache}/bin/elm-${ELM_VERSION}:$PATH" > $buildpack/export
```

Comment:
Maybe `>>` so this will continue to work if other things get added to the export script later?